apps/web/lib/features/activity/components/screenshoots-per-hour.tsx

```diff
@@ -26,60 +28,65 @@ export const ScreenshootPerHour = ({
 		},
 		[openModal]
 	);
-	return (
-		<div className="p-5 my-4 rounded-[1rem] space-y-5 dark:bg-[#1E2025] border-[.2rem] dark:border-[#FFFFFF0D] border-[#B993E6] ">
-			<h3 className=" font-medium">
-				{startedAt} - {stoppedAt}
-			</h3>
-			<div className="flex gap-5 flex-wrap ">
-				{timeSlots.map((el, i) => (
-					<ScreenshotItem
-						key={i}
-						endTime={el.stoppedAt}
-						startTime={el.startedAt}
-						percent={el.percentage}
-						imageUrl={el.screenshots[0]?.thumbUrl}
-						onShow={() => openScreenModal(el)}
-						idSlot={el.id}
-					/>
-				))}
-				<ScreenshotDetailsModal open={isOpen} closeModal={closeModal} slot={selectedElement} />
-			</div>
-		</div>
-	);
-};
 
-export const ScreenshootPerHourTeam = ({
-	timeSlots,
-	startedAt,
-	stoppedAt
-}: {
-	timeSlots: ITimerSlot[];
-	startedAt: string;
-	stoppedAt: string;
-}) => {
-	const { isOpen, closeModal, openModal } = useModal();
+	useEffect(() => {		
+		if (timeSlots && timeSlots.length > 0) {
+			console.log('First slot screenshots:', timeSlots[0]?.screenshots);
```

Comment:
@AnicetFantomas do we need this whole useEffect() (except for temp debugging)? Can we remove it? If yes, do in another PR based on develop branch as I will merge this PR with Squash.